server.go

```diff
@@ -0,0 +1,57 @@
+package main
```

Comment:
please follow the `go` structure 

https://github.com/golang-standards/project-layout
so, your server should be in internal package